website/docs/basics/block-manifest.md

```diff
@@ -85,3 +85,41 @@ If the `hasInnerBlocks` key is set to true, the block's `save` method for inner
 *custom feature*
 
 This key gives you the ability to use component attributes in your block without mapping all the component's attributes every time. Please check [this chapter](blocks-component-in-block) for more details.
+
+## The power of manifest.json
+
+As described before we use `manifest.json` to share stuff across multiple languages so you can easily see its power.
```

Comment:
```suggestion
As described before we use `manifest.json` to share stuff across PHP / JS / SCSS so you can easily see its power.
```

Reply:
Also: 

```As described before, we use `manifest.json` to share stuff across PHP / JS / SCSS so you can easily see its power.```

The comma after before word.